src/pages/team.js

```diff
@@ -1,12 +1,208 @@
 import React from 'react'; // required for Gatsby
 import { AppLayout } from '../layouts/AppLayout';
+import Card from 'react-bootstrap/Card';
+import Button from 'react-bootstrap/Button';
+import styles from "../styles/Box.css";
+
+import Container from 'react-bootstrap/Container';
+import Row from 'react-bootstrap/Row';
+import Col from 'react-bootstrap/Col';
+
+import Nishay from "../images/headshots/nsh.jpeg";
+import Ben from "../images/headshots/ben.jpeg";
+import Ivan from "../images/headshots/ivan.jpeg";
+import Shubh from "../images/headshots/shubh.jpeg";
+import Youssef from "../images/headshots/Youssef.jpeg";
+import Vasu from "../images/headshots/vasu.jpeg";
+import Nora from "../images/headshots/nora.jpeg";
+import Megan from "../images/headshots/meg.jpeg";
+import Alyson from "../images/headshots/ally.jpeg";
+import Jeremy from "../images/headshots/jeremy.jpeg";
+import David from "../images/headshots/david.jpeg";
+import Bernice from "../images/headshots/bernice.jpeg";
+import Luke from "../images/headshots/luke.jpeg";
+
+const cardInfo = [
+  {
+    description:
+      " Alyson is doing Master’s in Business Analytics (MSBA’22). She loves Music, food, and clean. Her favorite crypto is Cosmos",
+    title: "Alyson (Ran) Liu (Co-President)",
+    image: Alyson,
+    link: "https://www.linkedin.com/in/alysonliu/",
```

Comment:
Lets make link and object called links that is a list of links, where each link is {'type': 'url'}. Types would be LinkedIn, instagram, etc. 

Then in the render Card, iterate through the person's link(s) and display a clickable logo of the link type that will direct them to the page, rather than a generic 'Social' button